src/main/kotlin/racingcar/model/Winner.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar.model
+
+class Winner(private val racingGame: RacingGame) {
+
+    fun find(): String {
+        val farthestDistance = racingGame.getFarthestCarDistance()
+        return racingGame.racingCars.filter { it.movingDistance == farthestDistance }.joinToString { it.name }
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-racingcar/pull/55#discussion_r461527167

```suggestion
        return racingGame.racingCars.filter { it.isInPosition(farthestDistance) }.joinToString { it.name }
```

Reply:
수정하였습니다.